specification/network/resource-manager/Microsoft.Network/2017-06-01/expressRouteConnector.json

```diff
@@ -0,0 +1,396 @@
+		"version": "2017-06-01",
+		"title": "NetworkManagementClient"
+	},
+	"host": "management.azure.com",
+	"schemes":[
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security":[
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type":"oauth2",
+			"authorizationUrl":"https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow":"implicit",
+			"description":"Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation":"impersonate your user account"
+			}
+		}
+	},
+	"paths":{
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRouteConnectors/{expressRouteConnectorName}": {
+			"delete": {
+				"tags": [
+					"expressRouteConnectors"
+				],
+				"operationId": "expressRouteConnector_Delete",
+				"description": "Deletes the specified Express Route Connector Resource.",
+				"x-ms-examples": {
+                    "ExpressRouteConnectorDelete": { "$ref": "./examples/ExpressRouteConnectorDelete.json" }
+                },
+				"parameters": [
+					{
+						"$ref": "#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ExpressRouteConnectorNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/SubscriptionIdParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "Accepted and the operation will complete asynchronously."
+					},
+					"200": {
+						"description": "Delete successful."
+					},
+					"204": {
+						"description": "Delete successful."
+					},
+					"412": {
+						"description": "Precondition Failed"
+					},
+					"403": {
+						"description": "Forbidden"
+					}
+				},
+				"x-ms-long-running-operation": true
+			},
+			"get": {
+				"tags": [
+					"expressRouteConnector"
+				],
+				"operationId": "expressRouteConnector_Get",
+				"x-ms-examples": {
+                    "RouteFilterGet": { "$ref": "./examples/ExpressRouteConnectorGet.json" }
+                },
+				"description": "Gets the specfic Express Route Connector Resource from the specified resource group.",
+				"parameters": [
+					{
+						"$ref": "#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ExpressRouteConnectorNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/SubscriptionIdParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Request successful. The operation returns the Express Route Connector Resource resource.",
+						"schema": {
+							"$ref": "#/definitions/expressRouteConnector"
+						}
+					},
+					"404": {
```

Comment:
4XX status codes are not supported by autorest, please use default instead. You can do this however 
https://github.com/Azure/azure-rest-api-specs/blob/2fb9a0b3b902335ff0b0033711c234431931ec9d/specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/virtualNetworkRules.json#L55